common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -301,6 +316,10 @@ function () {
         this.videoAccessibleMenu.setValue(fileType);
         this.saveState(true, {'transcript_download_format': fileType});
         this.storage.setItem('transcript_download_format', fileType);
+
+        $(event.currentTarget).parent().parent().parent().next('.download-link')
```

Comment:
Yikes-- this seems incredibly brittle!! How can you do this without the great-grandparent reference?